app/views/govuk_publishing_components/components/docs/cards.yml

```diff
@@ -116,24 +116,30 @@ examples:
             text: Benefits
             path: http://www.gov.uk
           description: Includes eligibility, appeals, tax credits and Universal Credit
-  with_tracking_attributes:
+  with_data_attributes:
     description: |
-      Tracking can be enabled on the cards component by passing a minimum of data_track_category and data_track_action. Other tracking attributes are optional. Note: tracking events do not currently fire within the component guide.
+      Tracking can be enabled on the cards component by passing a minimum of `data_track_category` and `data_track_action`. Other tracking attributes are optional. Note: tracking events do not currently fire within the component guide.
```

Comment:
I think this is misleading - I believe tracking can't be enabled on the component unless it's wrapped in the `gemTrackClick` module? Suggest something simpler like:

```
Data attributes can be passed to individual links within the component as shown.
```

Reply:
Works for me.